rdock-utils/rdock_utils/common/types.py

```diff
@@ -14,10 +10,10 @@
 Matrix3x3 = numpy.ndarray[Any, numpy.dtype[numpy.float64]]
 SingularValueDecomposition = tuple[Matrix3x3, Vector3D, Matrix3x3]
 Superpose3DResult = tuple[CoordsArray, float, Matrix3x3]
+MatchIds = tuple[tuple[int, int], ...]
```

Comment:
This name may be a bit misleading, as it is not related to the match ids from sdtether (?)

Reply:
MappingIds perhaps?